libcxx/include/__ranges/view_interface.h

```diff
@@ -72,6 +73,27 @@ class view_interface {
     }
   }
 
+  constexpr auto cbegin()
+    requires input_range<_Derived>
```

Comment:
Ditto for [LWG3766](https://cplusplus.github.io/LWG/issue3766):
https://github.com/llvm/llvm-project/blob/445023f1739d3764f1dae6e6b7b0602328198a1e/libcxx/docs/Status/Cxx23Issues.csv#L223